components/global/drupal-form--default.vue

```diff
@@ -5,16 +5,60 @@
     v-bind="attributes"
     :action="useRoute().fullPath"
     class="drupal-form"
+    ref="form"
   >
     <slot><div v-html="content" /></slot>
   </form>
 </template>
 
 <script setup lang="ts">
-defineProps<{
+const props = defineProps<{
   formId: string
   attributes: object
   method: string
   content?: string
 }>()
+
+const form = ref(null)
+
+const unlockForm = () => {
+  const input = form.value?.querySelector('input[name="antibot_key"]');
+  if (input) {
+    input.value = props.attributes.dataAntibotKey.split("").reverse().join("").match(/.{1,2}/g).map((value) => value.split("").reverse().join("")).join("");
+    removeEventListeners();
+  }
+}
+
+const removeEventListeners = () => {
+  form.value?.removeEventListener('mousemove');
+  form.value?.removeEventListener('touchmove');
+  form.value?.removeEventListener('keydown');
+}
+
+if (props.attributes.dataAntibotKey) {
```

Comment:
we do not want specialized code like this in our general form helper. I think we need to find another way to solve this.